pootle/apps/pootle_translationproject/models.py

```diff
@@ -257,12 +261,14 @@ def save(self, *args, **kwargs):
                                       .get_or_make_subdir(self.project.code)
         self.pootle_path = self.directory.pootle_path
 
-        project_dir = self.project.get_real_path()
-        from pootle_app.project_tree import get_translation_project_dir
-        self.abs_real_path = get_translation_project_dir(
-            self.language, project_dir, self.file_style, make_dirs=not
-            self.directory.obsolete)
-
+        if not self.project.treestyle == "none":
```

Comment:
Nit: I find cleaner not to negate a condition when I can just switch the order in the branches.